core/src/main/scala/lucuma/sbtplugin/LucumaPlugin.scala

```diff
@@ -87,7 +87,11 @@ object LucumaPlugin extends AutoPlugin {
     )
 
     lazy val lucumaCoverageSettings = Seq(
-      coverageEnabled := githubIsWorkflowBuild.value, // enable in CI
+      coverageEnabled              := { // enable in CI, but only for the build job
+        githubIsWorkflowBuild.value && Option(System.getenv("GITHUB_JOB")).contains("build"),
```

Comment:
Btw, this also ensures that coverage will not be enabled for the deploy jobs in explore, etc. It is only running in the primary CI job for testing.